4.8/Dockerfile

```diff
@@ -6,15 +6,19 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    # Current keys from https://github.com/nodejs/node#release-team
```

Comment:
Ah, OK, this makes more sense when viewing the full change where you note two lists. But, see my other comments about having two lists. There's no need to add those older keys back since they were never used beyond v0.10 and v0.12